src/ncursesw.h

```diff
@@ -1607,14 +1607,6 @@ class NCursesWindow
 	return ::wtouchln( w, s, cnt, ( int )( changed ? 1 : 0 ) );
     }
 
-    /**
-     * Return TRUE if line is marked as changed, FALSE otherwise
-    */
-    bool	   is_linetouched( int line ) const
-    {
-	return ( ::is_linetouched( w, line ) ? TRUE : FALSE );
-    }
-
```

Comment:
BTW I have my own fix at https://github.com/libyui/libyui-ncurses/pull/61 . I do not check usage of this method, so removal is even better.  BTW what is reason for this ncursesw file which is almost copy of ncursesw file from ncurses itself?

Reply:
Probably just simplifying things. But I am not sure; that part goes back a long time to Michael Andres.